README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+# codacy-test
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion

### Among Codacy’s features
```